src/headless/converse-bookmarks.js

```diff
@@ -237,18 +237,17 @@ converse.plugins.add('converse-bookmarks', {
                         [__("The server did not return your bookmarks within the allowed time. "+
                             "You can reload the page to request them again.")]
                     );
-                } else {
-                    log.error('Error while fetching bookmarks');
-                    log.error(iq);
-                }
-                if (deferred) {
+                } else if (deferred) {
                     if (iq.querySelector('error[type="cancel"] item-not-found')) {
                         // Not an exception, the user simply doesn't have any bookmarks.
                         window.sessionStorage.setItem(this.fetched_flag, true);
                         return deferred.resolve();
                     } else {
                         return deferred.reject(new Error("Could not fetch bookmarks"));
```

Comment:
Here you still need to log an error though, otherwise we won't know what really went wrong.

Reply:
Updated accordingly.

Reply:
Thanks, sorry but the changelog entry needs to be moved to the top as well.

Reply:
Done